src/katsdpcalproc/pointing.py

```diff
@@ -236,23 +233,37 @@ def __call__(self, x):
         """
         return self._interp(x)
 
+# Helper function to calculate expected width
```

Comment:
It is better to have a proper function docstring than a comment above it in the code. That way we can access the help on the command line in IPython or in notebooks.